### PR TITLE
My branch

### PR DESCRIPTION
**Pull request by rozhi-98**
In this pull request:-

1. I have created a low wireframe UI with basic HTML and a little bit of CSS only to be able to demonstrate the adding and removing functionalities. 
2. I have created an array of objects in my javascript to hold the information about each book that I have to add or remove.
3. I have created the HTML of each book dynamically. Whenever correct information is written in the form and submitted, an event will be triggered and the inputs will be saved in an object inside the empty array.
4. The information will be saved in my local storage.

### CODE REVIEW

**Review by mckent05 — COMMENTED** [review]
Hi @rozhi-98,

# STATUS: INVALID REQUEST ❌ ❌ 
 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has:
- An empty PR summary ❌, please include a PR summary that summarises the changes you have made in this PR 👍 👍  
- A non-descriptive PR title 🚫 
- A vague branch name
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by rozhi-98** [comment]
Hi @mckent05 , thanks so much for reviewing my project. I will perform the changes you have required very soon, sorry for the inconvenience from my side.

File: app.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+    // Append book
+    appendBook(bookData);
+
+    // Adding to existing array
+    booksArray.push(bookData);
+
+    localStorage.setItem('books', JSON.stringify(booksArray));
+  });
+}
```
Comment (TedLivist) [inline app.js:73]: - [ ] **[OPTIONAL]** It would be nice to clear the inputs adding a book

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @rozhi-98,

✔️ Books can be added and removed as required

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.js (hunk 1 above)
Comment (rozhi-98) [reply app.js:73]: Thanks a lot for the review, I will take your suggestion into consideration and apply it before I proceed to the next milestone of the project.

**Pull request merged**